iconservice/iconscore/icon_score_context_util.py

```diff
@@ -60,16 +61,11 @@ def get_icon_score(context: 'IconScoreContext', address: 'Address') -> Optional[
 
     @staticmethod
     def _get_icon_score(context: 'IconScoreContext', address: 'Address') -> Optional['IconScoreBase']:
-        is_score_active = context.icon_score_deploy_engine.icon_deploy_storage.is_score_active(context, address)
-
-        if not is_score_active:
+        deploy_info = IconScoreContextUtil.get_deploy_info(context, address)
```

Comment:
Please add the type of deploy_info like the following:
deploy_info: 'DeployInfo' = IconScoreContextUtil.get_deploy_info(context, address)